sqs_workers/async_task.py

```diff
@@ -63,18 +62,18 @@ def run(self, *args: P.args, **kwargs: P.kwargs) -> Any:
             if len(args) > 0:
                 raise TypeError("Must use keyword arguments only for batch read queues")
             kwargs = bind_arguments(self.processor, [[kwargs]], {})
-            return self.processor(**kwargs)
+            return self.processor(**kwargs)  # type:ignore[call-arg]
         else:
             kwargs = bind_arguments(self.processor, args, kwargs)
-            return self.processor(**kwargs)
+            return self.processor(**kwargs)  # type:ignore[call-arg]
```

Comment:
I had to add this ignore even before making any changes. Running mypy failed with:

```
sqs_workers/async_task.py:65: error: Too few arguments  [call-arg]
sqs_workers/async_task.py:68: error: Too few arguments  [call-arg]
Found 2 errors in 1 file (checked 25 source files)
```

I did spent a couple of minutes trying to grasp the problem, but I couldn't. I'm not familiar with this codebase, so it may be more obvious to you. But if not, we can keep this for the time being. :)

Reply:
(same on line 65, it's the same problem)